tmpfiles.d/systemd.conf.m4

```diff
@@ -16,11 +16,6 @@ d /run/systemd/sessions 0755 root root -
 d /run/systemd/users 0755 root root -
 d /run/systemd/machines 0755 root root -
 d /run/systemd/shutdown 0755 root root -
-m4_ifdef(`ENABLE_NETWORKD',
-d /run/systemd/netif 0755 systemd-network systemd-network -
-d /run/systemd/netif/links 0755 systemd-network systemd-network -
-d /run/systemd/netif/leases 0755 systemd-network systemd-network -
-)m4_dnl
```

Comment:
hmm, i am not sure about this one... we have this API where people can install inotify watches on /run/systemd/netif/links to get notified about network config changes. Currently that works fine already relatively early at boot, as we create the dirs to watch with tmpfiles early on. But this is not going to work if we don't anymore...